input/pagecontent/iti-68.md

```diff
@@ -23,9 +23,14 @@ document from the Document Responder.
 
 <div>{% include MHD_ActorDiagram_ITI-68.svg %}</div>
 
-#### Retrieve Document Request Message
+####  Expected Actions
 
-#### Retrieve Document Response Message
+##### Federated Cross Community Access Option
+
+{% include iti-mhd-fxca-retrieve.html %}
+
+If the Document Responder supports the Federated Cross Community Access Option the Document Responder must be able to identify if a document retrieve 
+targets another community. If this is the case, it will need to forward the request to the target community, otherwise the request can be handled directly.
```

Comment:
Nitpick:

> otherwise the request can be handled directly

That's not what is shown in the diagram, the request is forwarded to another Document Responder in the same community.